app/models/collection.rb

```diff
@@ -14,13 +14,13 @@ class Collection < ApplicationRecord
   #   exercises << exercise unless exercises.find_by(id: exercise.id)
```

Comment:
Could you please also fix the `add_exercise` method (and change its name to `add_task`)? Please, also remember to re-enable the tests.

It's currently not used, but we should change that as the last step. I'll add some references to the old code (which you might copy and adjust) or the references still included) below.

**Routes:**
https://github.com/openHPI/codeharbor/blob/2e08353bbbcf3f33369d602c0f1e6eae8c849d4a/config/routes.rb#L108

**Controller:**
https://github.com/openHPI/codeharbor/blob/2e08353bbbcf3f33369d602c0f1e6eae8c849d4a/app/controllers/exercises_controller.rb#L8
https://github.com/openHPI/codeharbor/blob/2e08353bbbcf3f33369d602c0f1e6eae8c849d4a/app/controllers/exercises_controller.rb#L124-L131

**View (still included, just need to uncomment PARTS):** 
`app/views/tasks/show.html.slim`, lines 513-523

**Localizations (just need an update):**
`config/locales/en.yml`, lines 93-94 + line 675

**Tests (Model):**
`spec/models/collection_spec.rb`, line 74ff.

**Tests (Controller):**
https://github.com/openHPI/codeharbor/blob/2e08353bbbcf3f33369d602c0f1e6eae8c849d4a/spec/controllers/exercises_controller_spec.rb#L268-L276